fiatconnect-api.md

```diff
@@ -2233,6 +2234,37 @@ A KYC schema containing personal data about a user, as well as documents such as
 
 The `selfieDocument` and `identificationDocument` fields should be base64 encoded binary blobs representing images.
 
+#### 9.3.1.2. `PersonalDataAndDocumentsWithBack`
+
+A brasilian standard KYC schema containing personal data about a user, as well as documents such as an ID photo and selfie.
+
+```
+{
+	fullName: `string`,
+	dateOfBirth: {
+		day: `string`,
+		month: `string`,
+		year: `string`
+	},
+	address: {
+		address1: `string`,
+		address2?: `string`,
+		isoCountryCode: `string`,
+		isoRegionCode: `string`,
+		city: `string`,
+		postalCode?: `string`
+	},
+	phoneNumber: `string`,
+	idNumber: `string`,
+	email: `string`,
+	selfieDocument: `string`,
+	identificationDocumentFront: `string`,
+	identificationDocumentBack: `string`,
+}
+```
+
+The `selfieDocument`, `identificationDocument` and `identificationDocumentBack` fields should be base64 encoded binary blobs representing images.
```

Comment:
Can you provide some more detailed information on the fields required in this schema? (For example, how `phoneNumber` should be formatted, what the `idNumber` represents and how it should be formatted, etc.) It might seem a bit pedantic but it's important that we document the exact syntax of these fields.